src/systems/filecoin_mining/storage_mining/storage_miner_actor.id

```diff
@@ -106,7 +106,7 @@ type StorageMinerActorState struct {
     _updateSectorUtilization(rt Runtime, lastPoStResponse block.ChainEpoch) [deal.DealID]
     _initializeUtilizationInfo(rt Runtime, deals [deal.OnChainDeal]) sector.SectorUtilizationInfo
 
-    _getActivePower(rt Runtime) block.StoragePower
+    _getActivePower(rt Runtime, getActiveDealIDs bool) (activePower block.StoragePower, activeDealIDs [deal.DealID])
```

Comment:
this sort of breaks a software pattern regarding simplicity of `get`. I'd recommend splitting into two methods, calling one or two based on case (rather than passing in bool). Simpler

Reply:
+1

Reply:
@sternhenri I agree but in this case splitting into two will mean that we have to loop through the active sectors twice. Are we okay with that?

Reply:
Yes. I think that's fine. It's either two calls in the loop or one call in two loops. Now yes, the seek time for the object will be longer but leave to impl to optimize code. Let's make the spec as legible as possible

Reply:
sounds good